OSSM-2423: [DOC] must-gather tool needs to be more specific (#64534)
openshift/openshift-docs: modules/ossm-about-collecting-ossm-data.adoc

* The {product-title} CLI (`oc`) installed.

.Procedure

> . To collect {SMProductName} data with `must-gather`, you must specify the {SMProductName} image.

Reviewer: Not going to hold this PR up but adding a descriptor after `must-gather` could add clarity. I _think_ `must-gather` command would be right.

----
$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8:2.4 gather <namespace>
----

> 

Reviewer 2: If the list on lines 35 - 42 should be considered as part of step 2, add a plus sign (`+`) on line 34 to nest the list under the step.

> This creates a local directory that contains the following: 

Reviewer 2: ```suggestion
This creates a local directory that contains the following items:
```
IBM Style [Word Usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage) provides this guidance about the term _following_: "To introduce a list, procedure, or subsequent items, always use _following_ as an adjective before a noun. For example, do not write “Complete the following” or ”The functions include the following”. Instead, write “Complete the following steps” or “The adapter has the following functions”."

> * The Istio Operator namespace (and its children objects)

Reviewer 2: ```suggestion
* The Istio Operator namespace and its children objects
```
- Per [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-parentheses), "Avoid using parentheses in general text because text in parentheses is decreased in importance. Include information that is important and eliminate information that is not. Parentheses can also interrupt the flow of a sentence. Try to rewrite the text or use other punctuation instead of parentheses. If you must write a sentence that includes text in parentheses, ensure that the sentence is grammatically correct and technically accurate if you omit the parenthetical text."
- Should "children objects" be "child objects"?

> * All control plane namespaces (and their children objects)

Reviewer 2: ```suggestion
* All control plane namespaces and their children objects
```
Should "children objects" be "child objects"?

> * All namespaces (and their children objects) that belong to any service mesh

Reviewer 2: ```suggestion
* All namespaces and their children objects that belong to any service mesh
```
Similar comment here about "children objects"

* All Istio custom resource definitions (CRD)

> * All Istio CRD objects (VirtualServices in a given namespace, etc)

Reviewer 2: ```suggestion
* All Istio CRD objects, such as VirtualServices in a given namespace
```
Per IBM Style, the term _etc_ is a do-not-use term. Here's the guidance: "Do not use. Use _and so on_ when you list a clear sequence of elements, such as _1, 2, 3, and so on_ or _Monday, Tuesday, Wednesday, and so on_. When you list items that do not form a clear sequence, use appropriate descriptive wording, such as _device drivers, firmware, and other code_."
